eth-rpc-adapter/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:14-alpine
+
+ENV NODE_ENV=production
+
+WORKDIR /usr/src/app
+
+COPY . .
+COPY docker/package.json ./package.json
+COPY docker/yarn.lock ./yarn.lock
+
+RUN yarn
+RUN [ "yarn", "build" ]
+
+EXPOSE 8545
+
```

Comment:
is it kind of incoming port or just connecting to external world ?

Reply:
this will be exposed to external world